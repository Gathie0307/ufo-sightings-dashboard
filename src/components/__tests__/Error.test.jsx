import { render } from '@testing-library/react';
import Error from '../Error';

describe('Error', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(
      <Error title='Invalid Data' message='Network Error' />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
