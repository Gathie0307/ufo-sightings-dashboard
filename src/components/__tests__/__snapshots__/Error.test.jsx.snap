// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Error > should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex justify-center absolute top-55 left-80 "
  >
    <div
      class="py-1"
    >
      <svg
        class="fill-current h-10 w-10 text-red-600 mr-4"
        viewBox="0 0 20 20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M2.93 17.07A10 10 0 1 1 17.07 2.93 10 10 0 0 1 2.93 17.07zm12.73-1.41A8 8 0 1 0 4.34 4.34a8 8 0 0 0 11.32 11.32zM9 11V9h2v6H9v-4zm0-6h2v2H9V5z"
        />
      </svg>
    </div>
    <div>
      <p
        class="text-xl font-bold text-red-600"
      >
        Invalid Data
      </p>
      <p
        class="text-sm text-gray-600"
      >
        Network Error
      </p>
    </div>
  </div>
</DocumentFragment>
`;
